test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManagement.cs

```diff
@@ -62,7 +62,33 @@ public string[] GetInstallCommandsFromLocal(string path)
 
         public string[] GetInstallCommandsFromMicrosoftProd(Option<Uri> proxy)
         {
-            throw new NotImplementedException("aziot-edge and aziot-identity-service currently aren't available in package repos");
+            string debPackages = $"https://packages.microsoft.com/config/{this.os}/{this.version}/multiarch/prod.list";
+            if (this.os.ToLower() == "ubuntu" && this.version == "20.04")
+            {
+                // Should be changed to 20.04 when we publish to that repo.
+                debPackages = $"https://packages.microsoft.com/config/{this.os}/18.04/multiarch/prod.list";
+            }
+            return this.packageExtension switch
+            {
+                SupportedPackageExtension.Deb => new[]
+                {
+                    $"curl {debPackages} > /etc/apt/sources.list.d/microsoft-prod.list",
+                    "curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /etc/apt/trusted.gpg.d/microsoft.gpg",
+                    $"apt-get update",
+                    $"apt-get install --yes aziot-edge"
+                },
+                SupportedPackageExtension.Rpm => new[]
+                {
+                    $"rpm -iv --replacepkgs https://packages.microsoft.com/config/{this.os}/{this.version}/packages-microsoft-prod.rpm",
+                    $"yum updateinfo",
+                    $"yum install --yes aziot-edge",
+                    "pathToSystemdConfig=$(systemctl cat aziot-edge | head -n 1)",
+                    "sed 's/=on-failure/=no/g' ${pathToSystemdConfig#?} > ~/override.conf",
+                    "sudo mv -f ~/override.conf ${pathToSystemdConfig#?}",
+                    "sudo systemctl daemon-reload"
+                },
+                _ => throw new NotImplementedException($"Don't know how to install daemon on for '.{this.packageExtension}'"),
+            };
```

Comment:
Do we have any tests to just verify we got correct commands?

Reply:
I did only manual testing for Ubuntu. Can't imagine how you can automate that and if it worth it. Also, this code will execute only on dev box, so pipeline is not affected as well.

Reply:
Was more thinking a UT to just verify the strings :P but your call on whether it adds any value, as you said this is for dev purposes only